galley/pkg/envvar/envvars.go

```diff
@@ -43,6 +43,9 @@ var (
 
 	// MCPSourceReqFreq is the frequency that is used by the rate limiter in MCP Sources
 	MCPSourceReqFreq = env.RegisterDurationVar("MCP_SOURCE_REQ_FREQ", time.Second, "")
+
+	// EnableIncrementalMCP is an option to enable incremental mcp
+	EnableIncrementalMCP = env.RegisterBoolVar("ENABLE_INCREMENTAL_MCP", false, "")
```

Comment:
can enable it by default, as this is a restore of previous feature?

Reply:
I think it would be more rigorous to disable by default.